specification/applicationinsights/resource-manager/microsoft.insights/preview/2017-10-01/componentFeaturesAndPricing_API.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
This one looks like the one in https://github.com/Azure/azure-rest-api-specs-pr/pull/275/files